analyzer/codechecker_analyzer/analyzer.py

```diff
@@ -181,9 +181,16 @@ def perform_analysis(args, skip_handlers, actions, metadata_tool,
         missing_checkers = checkers.available(args.ordered_checkers,
                                               available_checkers)
         if missing_checkers:
-            LOG.error("No checker(s) with these names was found:\n%s",
-                      '\n'.join(missing_checkers))
-            sys.exit(1)
+            diag_msg = "No checker(s) with these names was found:\n{}".format(
+                '\n'.join(missing_checkers))
+            if 'no_missing_checker_error' in args:
+                LOG.warning(diag_msg)
+            else:
+                LOG.error(diag_msg)
+                LOG.info("If you want to suppress errors relating to unknown "
```

Comment:
Maybe include the recommendation from the commit message here too, that it is not advised to use this feature.

Reply:
I've considered this, and it usually is my jam to shove my will down on anyones throat, but I figured that any one person that switches an error to a warning must know what they are doing.